node/src/cli.rs

```diff
@@ -109,6 +109,12 @@ pub struct ProviderConfigurations {
     /// Extrinsic retry timeout in seconds.
     #[clap(long, default_value = "60")]
     pub extrinsic_retry_timeout: u64,
+
+    /// MSP charging fees frequency.
```

Comment:
You might want to add more info like the unit of this parameter. For example, is it in number of blocks? And if so, could be helpful to give some parameter like "Setting it to 600, with 6 seconds blocks, means charging every 1 hour".

Also, wouldn't this be a _period_ instead of a frequency?

Reply:
Yes period is better. I changed every 'freq' with 'period'.